bot/university/university.py

```diff
@@ -210,8 +208,8 @@ async def exchange_error(self, ctx: commands.Context, error: commands.CommandErr
                 delete_after=15.0)
         if isinstance(error, commands.CommandInvokeError) and isinstance(error.original, IntegrityError):
             await ctx.channel.send(
-                "**__Error:__** Du hast für diesen Kurs bereits eine Tauschanfrage aktiv. Du kannst sie mit "
-                "'exchange remove <channel-mention> löschen.'", delete_after=15.0)
+                "**__Error:__** Du hast für diesen Kurs bereits eine Tauschanfrage aktiv. Du kannst sie mit `"
+                "exchange remove <channel-mention>` löschen.", delete_after=15.0)
         if isinstance(error, commands.CommandInvokeError) and isinstance(error.original, SyntaxError):
             await ctx.channel.send(
```

Comment:
Missing prefixes and the error messages should be send via direct message to the user. Posting them in the exchange channel just confuses the other members.